docs/src/test-sharding-js.md

```diff
@@ -62,20 +71,20 @@ jobs:
       run: npx playwright install
 
     - name: Run Playwright tests
-      run: npx playwright test --shard ${{ matrix.shard }}/4
+      run: npx playwright test --shard ${{ matrix.shard }}/4 --reporter blob
```

Comment:
We mentioned above that one has to add 'blob' reporter to the config. So I'd rather drop this parameter here as it is not what we ourselves do.